addons/overheating/stringtable.xml

```diff
@@ -144,7 +144,25 @@
             <Chinese>過熱的武器將會有打不準和減少射擊初速的情況。適用於所有玩家</Chinese>
             <Hungarian>A túlmelegedett fegyverek kevésbé lesznek pontosak és csökkent a lövés sebessége. Minden játékosra vonatkozik.</Hungarian>
         </Key>
-        <Key ID="STR_ACE_Overheating_unJamOnreload_displayName">
+        <Key ID="STR_ACE_Overheating_particleEffectsAndDispersionDistance_displayName">
+            <English>Distance for Effects and Dispersion</English>
+        </Key>
+        <Key ID="STR_ACE_Overheating_particleEffectsAndDispersionDistance_description">
+            <English>Overheating particle effects and dispersion will only be show for other player units closer than this. Value is in metres.</English>
+        </Key>
+        <Key ID="STR_ACE_Overheating_overheatingRateOfFire_displayName">
+            <English>Increase rate of fire with heat</English>
```

Comment:
Does it fit the UI? Maybe `Heat increases ROF`? @mharis001

Reply:
Also fits on my screen.

Reply:
It's shorter than `STR_ACE_Overheating_showParticleEffectsForEveryone_displayName` above.

Reply:
I like @jonpas 's suggestion of "Heat Increases ROF". I think it reads more naturally.